src/Moderators.svelte

```diff
@@ -0,0 +1,118 @@
+          </button>
+          {#if data.name == 'Stephen Mount'}
+          <button
+            class="bg-purple-400 rounded px-1 text-white text-sm ml-auto"
+          >
+            Server Booster
+          </button>
+          {/if}
+          <!--individul ifs or group if both can be done :)-->
+          {#if data.name=='Michael Mba' || data.name=='Eddie Jaoude'}
+          <button class="bg-green-400 rounded px-1 text-white text-sm ml-auto">
+          {data.name=='Michael Mba'?'Python':'Opensource'}
+          </button>
+          {/if}
+          <h1
+            class="font-medium text-lg sm:text-xl md:text-xl lg:text-xl xl:text-xl font-medium pt-2 sm:pt-1 md:pt-1 lg:pt-1"
+          >
+           {data.name}
+          </h1>
+          <p
+            class="font-light text-xs sm:text-sm md:text-sm lg:text-sm xl:text-sm py-1 font-normal hidden md:block lg:block break-words"
+          >
+          {data.desc}
+          </p>
+          <div
+            class="hidden lg:inline-flex md:inline-flex flex flex-inline"
+          >
+            <div class="flex flex-inline">
+              <a target="_blank" href={data.github}>
+                <i
+                  class="fa fa-github-alt pr-1 pt-6"
+                  style="
+                    font-size: 18px;
+                    color: rgb(109, 71, 199);
+                    font-style: normal;
+                  "
+                ></i>
+                <p
+                  class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
+                >
+                  GitHub
+                </p>
+              </a>
+            </div>
+            <div class="flex flex-inline">
+              <a target="_blank" href={data.website}>
+                <i
+                  class="fa fa-link pr-1 pt-6 text-purple-500"
+                  style="
+                    font-size: 18px;
+                    font-style: normal;
+                  "
+                ></i>
+                <p
+                  class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
+                >
+                  Website
+                </p>
+              </a>
+            </div>
+          </div>
+          <!-- for small screens -->
+          <div
+            class="flex-inline lg:hidden sm:flex-inline md:hidden flex flex-inline"
+          >
+            <div>
+              <a target="_blank" href={data.github}>
+                <i
+                  class="fa fa-github-alt pt-5 pr-5"
+                  style="font-size: 20px; color: rgb(109, 71, 199)"
+                ></i>
+              </a>
+            </div>
+            <div>
+              <a target="_blank" href={data.website}>
+                <i
+                  class="fa fa-link pt-5 pr-5"
+                  style="font-size: 20px; color: rgb(109, 71, 199)"
+                ></i>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  {/each}
+</div>
+</div>
```

Comment:
are these two divs necessary? They are on the same level of indentation 😅 so it's a bit confusing. I think what is not indented is the content of the grid.
This is not a blocker for the PR, simply a comment to keep in mind later.

Reply:
WIll keep it in mnd for sure